### ISSUE TITLE
Traceback from !rss add https://www.irozhlas.cz/rss/irozhlas

### ISSUE THREAD

**Issue by Mikaela** [bitbot-irc/bitbot#85]
```
Jul 08 16:10:08 relpda python3[8127]: 2019-07-08T16:10:08.490Z [INFO] pirateirc#cz-feed | <@MikaelaSuomalainen> !rss add https://www.irozhlas.cz/rss/irozhlas                                                                                                                                       Jul 08 16:10:10 relpda python3[8127]: 2019-07-08T16:10:10.692Z [ERROR] failed to call event "received.message.channel"                            Jul 08 16:10:10 relpda python3[8127]: Traceback (most recent call last):                                                                          Jul 08 16:10:10 relpda python3[8127]:   File "/home/bitbot/bitbot/src/EventManager.py", line 169, in _call                                        Jul 08 16:10:10 relpda python3[8127]:     returned = hook.call(event)                                                                             Jul 08 16:10:10 relpda python3[8127]:   File "/home/bitbot/bitbot/src/EventManager.py", line 41, in call                                          Jul 08 16:10:10 relpda python3[8127]:     return self.function(event)                                                                             Jul 08 16:10:10 relpda python3[8127]:   File "/home/bitbot/bitbot/modules/commands/__init__.py", line 289, in channel_message                     Jul 08 16:10:10 relpda python3[8127]:     command_prefix=command_prefix)                                                                          Jul 08 16:10:10 relpda python3[8127]:   File "/home/bitbot/bitbot/modules/commands/__init__.py", line 234, in command                             Jul 08 16:10:10 relpda python3[8127]:     hook.call(new_event)                                                                                    Jul 08 16:10:10 relpda python3[8127]:   File "/home/bitbot/bitbot/src/EventManager.py", line 41, in call                                          Jul 08 16:10:10 relpda python3[8127]:     return self.function(event)                                                                             Jul 08 16:10:10 relpda python3[8127]:   File "/home/bitbot/bitbot/modules/rss.py", line 121, in rss
Jul 08 16:10:10 relpda python3[8127]:     seen_ids = self._check_url(url)
Jul 08 16:10:10 relpda python3[8127]:   File "/home/bitbot/bitbot/modules/rss.py", line 94, in _check_url                                         Jul 08 16:10:10 relpda python3[8127]:     return [entry["id"] for entry in feed["entries"]]
Jul 08 16:10:10 relpda python3[8127]:   File "/home/bitbot/bitbot/modules/rss.py", line 94, in <listcomp>                                         Jul 08 16:10:10 relpda python3[8127]:     return [entry["id"] for entry in feed["entries"]]
Jul 08 16:10:10 relpda python3[8127]:   File "/home/bitbot/.linuxbrew/Cellar/python/3.7.3/lib/python3.7/site-packages/feedparser.py", line 356, i$ __getitem__                                                                                                                                      Jul 08 16:10:10 relpda python3[8127]:     return dict.__getitem__(self, key)
```

There isn't even an error about failure to add the feed, only this in terminal

**Comment by jesopo** [comment]
turns out, RSS entries having an `id` part is **optional**. not sure what I'll use otherwise to determine things that are new.... maybe `<link>`?

**Comment by jesopo** [comment]
should be resolved as of jesopo/bitbot@33e462d27b810ffe0b9360e33625da2b79e97153
